m3u8/__init__.py

```diff
@@ -15,14 +15,16 @@
     from urllib2 import urlopen, Request, HTTPError
     from urlparse import urlparse, urljoin
 
-from m3u8.model import M3U8, Playlist, IFramePlaylist, Media, Segment, PartialSegment, PartialSegmentList, PartInformation, RenditionReport, ServerControl
+from m3u8.model import M3U8, Segment, SegmentList, PartialSegment, PartialSegmentList, Key, Playlist, IFramePlaylist, Media, MediaList, PlaylistList, Start, RenditionReport, RenditionReportList, ServerControl, Skip, PartInformation
```

Comment:
This line looks a bit long. Could it be improved using parentheses?

Reply:
Yes, fair point. I'll make the change later.

Reply:
Thank you!!!

Reply:
Fixed via bd764b3